pkg/cvo/internal/operatorstatus.go

```
@@ -97,6 +99,13 @@ func (b *clusterOperatorBuilder) Do(ctx context.Context) error {
 	if b.modifier != nil {
 		b.modifier(os)
 	}
+	if b.mode == resourcebuilder.PrecreatingMode {
+		_, err := b.createClient.Create(os)
+		if err != nil && kerrors.IsAlreadyExists(err) {
+			err = nil
+		}
+		return err
```

after a successful create, need to `UpdateStatus` and ignore any errors.